source/process/sg_mattermost-doc-style.rst

```diff
@@ -0,0 +1,28 @@
+====================================
+Mattermost Documentation Style Guide
+====================================
+
+This is the Mattermost style guide for documentation. It acts as a reference for writers and editors to ensure that the Mattermost documentation is consistent and clear.
+
+.. Note::
+  The style guide is not intended to slow down or otherwise impede contributions, which are always welcome. No contribution will be rejected due to non-conforming style, although it might be edited.
+
+The Mattermost documentation must be of high quality. It must be accurate and clear, and be presented with a style and tone that is appropriate for technical content. People who use Mattermost rely on the documentation to get their jobs done. We don't want to see an installation of Mattermost delayed because the documentation has an error or is difficult to understand.
+
+.. comments
+
+  Main screen, Navigation panel, Message Details panel
```

Comment:
Er... kind of confused by this section...
![image](https://cloud.githubusercontent.com/assets/177788/19996141/a585cd76-a21a-11e6-8dec-1a612037822a.png)